CONTRIBUTING.md

```diff
@@ -36,6 +36,7 @@ To simplify this, it is possible to:
 - iterate on ZIM UI code
 
 To achieve this, first build the Docker image based on current code base.
+Make sure Docker is running in the background ("Docker Deamon" app for Windows and Mac, or `docker start` in Linux).
```

Comment:
This should not be placed here, we should instead rewrite the `contributions` section:
- state the requirements more clearly: Python 3.12 and preferably Docker (which engine must run in the background)
- add mention of `pre-commit` which must be activated (code must pass all pre-commit checks to succeed in the CI)

Reply:
Done